### DISCUSSION TITLE
Can an attacker destroy Chia when they have > 51% total size of plots? currently ~60PB

### DISCUSSION THREAD

**Discussion by Stumble** [Chia-Network/chia-blockchain#1407] [Q&A]
Just had a fullnode and a plot up and running, exciting project!

Without a deep dive into the green paper, I have a naive question: could an attacker control Chia when he possess more than a half of the network space? Currently the estimated network space is 120PB. I assume it's the total size of all plots. Let's say an attacker creates 60PB plots and joins the network, what damage could he do?

**Comment by hoffmang9** [comment]
Just like any Nakamoto consensus, an attacker can do lots of arbitrary stuff like double spends by controlling 51% or more of the resource. On 120PiB that means that an attacker would need 121PiB additional assuming no more non attacker space came on. 1.1 PiB more from not attackers would take safety back if the attacker only held 121PiB.

**Reply by Stumble** [reply]
> Just like any Nakamoto consensus, an attacker can do lots of arbitrary stuff like double spends by controlling 51% or more of the resource. On 120PiB that means that an attacker would need 121PiB additional assuming no more non attacker space came on. 1.1 PiB more from not attackers would take safety back if the attacker only held 121PiB.

Thanks for answering. Yeah I was wrong with those numbers. Assuming all 120PiB are controlled by honest nodes now, attackers need 121PiB more to do arbitrary stuff. 

My concern is that, it seems that possesing more computation power (work) is much harder than plots (space). Presumably, harddrives are generally cheaper than computing chips, is there any related analysis in the green paper?

**Reply by hoffmang9** [reply]
The size of the market for PoW ASICs is tiny compared to the Zetabyte and ~$90B spent buying storage last year. As such it is REALLY hard to corner the storage market where the same money spent in the PoW ASIC market is huge. At scale there is a lot of stranded and unused storage that an attacker will end up competing against.

Note that there are already about 8000 public chia nodes in the first couple of days of the network. That is the other big protection. Send three elements of troops out in the right country right now and you can 51% attack BTC.

**Reply by Stumble** [reply]
> The size of the market for PoW ASICs is tiny compared to the Zetabyte and ~$90B spent buying storage last year. As such it is REALLY hard to corner the storage market where the same money spent in the PoW ASIC market is huge. At scale there is a lot of stranded and unused storage that an attacker will end up competing against.

I see. It would be awesome if there could be an analysis of this aspect somewhere in the doc. I suspect that there are more people, not just me, have concerns about it.

> Send three elements of troops out in the right country right now and you can 51% attack BTC.

I get your point, lol, but sending three elements of troops out to that country might just end this world:) :bomb:

**Reply by hoffmang9** [reply]
Not if it's the host countries own troops... And that is what makes it an extra serious problem. Hard to hide a large ASIC install. Easy to blend in with a bunch of storage in a data center.

**Reply by dorkmo** [reply]
Easier to hide at home too! much quieter than a bunch of graphics cards crying for mercy

**Reply by danieljperry** [reply]
Here's my slightly different way of looking at it.

How to 51% attack a PoS chain:
1. Raise money (easy to do if you are a large rogue nation).
2. Buy a 51% stake.
3. Profit!
Time required to pull off this attack: Very little. Likely has been done already.

How to 51% attack Bitcoin (PoW):
1. Raise money (easy to do if you are a large rogue nation).
2. Use the money to buy more ASIC miners than currently exist in the world. Install and activate all of this equipment without causing a major power outage.
3. Profit!
Time required: Some, considering the ASIC supply would have to catch up to meet demand.

How to 51% attack Chia (PoST):
1. Raise money (easy to do if you are a large rogue nation).
2. Use the money to buy high-end servers, SSDs, and HDDs.
3. (This is the key) Use the high-end servers and SSDs to create plots on the HDDs.
4. Profit!
Time required: A lot. For now it may be possible to get your hands on the equipment, but you would still need to plot it at a greater rate than everyone else combined. And after Chia's capacity grows by, say, 100x, it will be difficult to acquire the amount of storage required to pull off the attack. And it will be much, much more difficult to create plots at a pace that everyone else couldn't keep up with.

In other words, 51% attacking a PoST chain requires one extra step beyond what is required to attack a PoW chain, and two extra steps beyond what is required to attack a PoS chain.
